README.md

```diff
@@ -621,6 +622,39 @@ api.PathPart = "users/abc";
 await api.GetAsync();
 ```
 
+HTTP Request Message Properties
```

Comment:
Last minor thing: can we put this at the bottom of the "Controlling the Requests" section? I think it's worth introducing the concept after we're introduced `RequestModifier` and custom `HttpClientHandlers` - it doesn't make sense without knowing about those.